docs/configuration.rst

```diff
@@ -11,33 +11,33 @@ Cache
-| hadoop.cache.data.dirsuffix              | The name of the subdirectory to be used for storing cache files.       | string            | /fcache/         | C & S         |
+| rubix.cache.dirsuffix                    | The name of the subdirectory to be used for storing cache files.       | string            | /fcache/         | C & S         |
 |                                          | Example: /media/ephemeral0\ **/fcache/**                               |                   |                  |               |
 +------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
-| hadoop.cache.data.expiration.after-write | The time files will be kept in cache prior to eviction.                | integer (ms)      | MAX_VALUE        | S             |
+| rubix.cache.expiration.after-write       | The time files will be kept in cache prior to eviction.                | integer (ms)      | MAX_VALUE        | S             |
 +------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
-| hadoop.cache.data.fullness.percentage    | The percentage of the disk space that will be filled with cached data  | integer (%)       | 80               | S             |
+| rubix.cache.usage.percentage             | The percentage of the disk space that will be filled with cached data  | integer (%)       | 80               | S             |
 |                                          | before cached files will start being evicted.                          |                   |                  |               |
 +------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
-| hadoop.cache.data.strict.mode            | Propagate exceptions if there is an error while caching data if true;  | boolean           | false            | C             |
+| rubix.cache.strict.mode                  | Propagate exceptions if there is an error while caching data if true;  | boolean           | false            | C             |
 |                                          | otherwise fall back on reading data directly from remote file system   |                   |                  |               |
 +------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
-| rubix.enable.file.staleness-check        | When true, always check for updates to file metadata from remote       | boolean           | true             | S             |
+| rubix.cache.enable.file.staleness-check  | When true, always check for updates to file metadata from remote       | boolean           | true             | S             |
```

Comment:
Please move **enable** to the end of the config-name like **rubix.cache.file.staleness-check.enable**